src/components/Movies.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import "../App.css";
+
+
+class Movies extends React.Component {
+    constructor() {
+        super()
+
+        this.state = {
+            movie: [],
+            movieTitle: null,
+            movieDate: null,
+            movieDescrip: null
+        }
+    }
+
+    fetchPeople = () => {
+        fetch('https://ghibliapi.herokuapp.com/films')
+        .then((res) => res.json())
+        .then((data) => {
+            this.setState({
+                movie: data,
+            })
+        })
+    }
```

Comment:
```suggestion
    fetchMovie = () => {
        fetch('https://ghibliapi.herokuapp.com/films')
        .then((res) => res.json())
        .then((data) => {
            this.setState({
                movie: data,
            })
        })
    }
```
In `componentDidMount`, you're calling a function called `this.fetchMovie()`. It's a small typo, but it's keeping you from calling your function that fetches your data and sets it in state, which is a huge deal.